src/js/dom7-utils.js

```diff
@@ -1,6 +1,6 @@
-// DOM Library Utilites
+// DOM Library Utilities
 $.parseUrlQuery= function (url) {
-    var url = url || location.href;
+    url = url || location.href;
```

Comment:
@ZanderBrown `url` is a string so it won't be modified as it passed by value not by the link